docs/site/Calling-soap-web-services.md

```diff
@@ -0,0 +1,191 @@
+---
+lang: en
+title: 'Calling SOAP Web services'
```

Comment:
Making the title to have the first character of each word to be upper case (consistent with other docs).
```suggestion
title: 'Calling SOAP Web Services'
```

Reply:
Done